database/vertica/vertica.go

```diff
@@ -79,16 +83,20 @@ func (v *Vertica) Close() error {
 	return nil
 }
 
-
 func (v *Vertica) dropSchema() error {
-	query := fmt.Sprintf("DROP SCHEMA %s CASCADE",  v.config.Schema)
+	query := fmt.Sprintf("DROP SCHEMA %s CASCADE", v.config.Schema)
 	_, e := v.conn.QueryContext(context.Background(), query)
 	return e
 }
 
-
 func (v *Vertica) dropTables() error {
-	query := `SELECT TABLE_NAME FROM v_catalog.tables`
+
+	schema, err := v.configuredSchema()
+	if err != nil {
+		return err
+	}
+	query := fmt.Sprintf(`SELECT TABLE_NAME FROM v_catalog.tables WHERE TABLE_SCHEMA = '%s'`, schema)
```

Comment:
Have you used [`Lex`](https://godoc.org/github.com/vertica/vertica-sql-go/parse#Lex) before? I'm wary of quoting parameters ourselves.